cfme/modeling/base.py

```diff
@@ -190,3 +190,18 @@ def parent_of_type(obj, klass):
     for x in _walk_to_obj_root(obj):
         if isinstance(x, klass):
             return x
+
+
+def match_items(items, filter):
+    ''' Return only items matching filter dict. '''
```

Comment:
double quotes on these docblocks please

Reply:
Consider making this a BaseCollection classmethod instead of just this standalone method, for clarity during use and ability to call `self.match_items(self.all(), self.filters)`